cypress/integration/home-page.spec.js

```diff
@@ -0,0 +1,37 @@
+/// <reference types="Cypress" />
+
+context('Home Page', () => {
+  beforeEach(() => {
+    cy.visit('http://localhost:9090');
+  });
+
+  it('should load wikipedia cube', () => {
+    cy.get('.cube-header-bar .title')
+      .should('contain', 'Wikipedia Example');
+  });
```

Comment:
Is `cy.get` promise? 

Does Cypress require to return `cy` when async/await notation is not used?
```js
it('should load wikipedia cube', () =>  cy
  .get('.cube-header-bar .title')
  .should('contain', 'Wikipedia Example'));
```

Reply:
It is something more complicated. It runs multiple times so more like a stream. Anyways, I shouldn't return them - I will get warning in console. And awaiting on them is also not good practice (but possible) so I rewrote last test.

Reply:
👍